simulation/array_simulator.pro

```diff
@@ -162,7 +182,7 @@ PRO array_simulator,vis_arr,vis_weights,obs,status_str,psf,params,jones,error=er
   IF Keyword_Set(snapshot_healpix_export) THEN begin
     IF ~Keyword_Set(n_avg) THEN n_avg=1
     IF Keyword_Set(snapshot_recalculate) THEN status_str.healpix_cube=(status_str.hpx_even=(status_str.hpx_odd=0))
-    healpix_snapshot_cube_generate,obs,status_str,psf,cal,params,vis_arr,/restrict_hpx_inds,$
+    healpix_snapshot_cube_generate,obs,status_str,psf,cal,params,vis_arr,hpx_radius=Float(select_radius),$
```

Comment:
Removing the `/restrict_hpx_inds` keyword will change the behavior of `array_simulator` for all other users. We perhaps don't want that hardcoded, but it should still be passed in and kept as the default.

Reply:
`select_radius` should be returned as a `Float` from the (new) function that calculates it. The extra conversion to `Float` is unnecessary and confusing.